installation/scripts/kyma-scripts/testing-common.sh

```diff
@@ -0,0 +1,182 @@
+ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
Remove it and use kyma-cli

Reply:
Discussed on site

Reply:
Okay, now I know that we can't use it because it calls `helm test` underneath. Please create GitHub issue for Kyma CLI about that.